client/lua/crayon.lua

```diff
@@ -0,0 +1,342 @@
+local requests = require("requests")
+local socket = require("socket")
+local json = require('cjson.safe')
+
+local __version__ = "0.4"
+
+local CrayonClient = {}
+local CrayonExperiment = {}
+
+-- CrayonClient class
```

Comment:
I wasn't sure which class system to use and frankly none was really needed here given that we have 2 classes.